2func.py

```diff
@@ -0,0 +1,5 @@
+def f(list):
+    list.sort(key=lambda list: len(list))
```

Comment:
You change input data, think why this is not good, may be better to use another way?